datalake/src/main/java/com/sequenceiq/datalake/service/sdx/stop/SdxStopService.java

```diff
@@ -65,9 +79,18 @@ public class SdxStopService {
     @Inject
     private RegionAwareInternalCrnGeneratorFactory regionAwareInternalCrnGeneratorFactory;
 
+    @Inject
+    private FlowLogService flowLogService;
+
+    @Inject
+    private FlowChainLogService flowChainLogService;
+
     public FlowIdentifier triggerStopIfClusterNotStopped(SdxCluster cluster) {
         MDCBuilder.buildMdcContext(cluster);
         freeipaService.checkFreeipaRunning(cluster.getEnvCrn());
+        if (!checkIfStoppable(cluster)) {
+            throw new CloudbreakServiceException("Cannot stop datalake!");
```

Comment:
Can you please change the error message to something that describes why the datalake can't be stopped?

Reply:
Yeah I'll make this a little less vague.